proposed/http-client/http-client-meta.md

```diff
@@ -77,8 +76,8 @@ the exception handling in a consuming library would in most cases not be differe
 
 The initial idea was to allow the client to be configured to throw exceptions for responses
 with HTTP status 4xx and 5xx. That approach is not desired because consuming libraries would
-have to check for 4xx and 5xx responses twice. First by verifying the status code on the response and
-then by catching possible exceptions.
+have to check for 4xx and 5xx responses twice: first, by verifying the status code on the response,
+and second by catching potential exceptions.
```

Comment:
This was a continuation, so I used a colon. Additionally, since you began with "first", the second point required a "second".